src/workers/hermes_strategies/regular_interval_upload_strategy.js

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import HermesUploadStrategy from './upload_strategy';
+import {getTimestamp} from '../../utils/time_utils';
+
+export default class RegularIntervalUploadStrategy extends HermesUploadStrategy {
+  constructor() {
+    super();
+    this.lastBundlingTimestamp = null;
+  }
+
+  bundlingPeriod() {
+    return 2000;
```

Comment:
Isn't 2000 too small with block time 5 seconds?

Reply:
The consequence would be the fact that there will be more than one bundle added during one block. Bundling frequency should be dependent on traffic generated on node, not on the blockchain details. Also this is only sample strategy, so I haven't really analysed what is correct in business way and what is not. Specific strategies to be added in another ticket(s)